prophet-ressa/src/lib.rs

```diff
@@ -1,8 +1,30 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        let result = 2 + 2;
-        assert_eq!(result, 4);
+mod gen_ressa;
+
+use gen_ressa::*;
+use source_code_parser::{ModuleComponent, ressa::{RessaResult, run_ressa_parse}};
+
+/// Errors that arise while retrieving entity/endpoint pairs
+#[ derive() ]
+pub enum Error {
+    IOError(std::io::Error),
+    MinifyError(prophet_ressa_minify::MinifyError),
+}
+
+/// Generate a string for displaying the error
+impl From<Error> for String {
+    fn from(err: Error) -> Self {
+        match err {
+            Error::IOError(io_err) => format!("IO error: {}", io_err),
+            Error::MinifyError(min_err) => format!("Minify error: {}", min_err),
+        }
     }
 }
```

Comment:
I think we could derive Debug and thiserror::Error here (see the minify crate for an example)